python-modules/cis_profile/cis_profile/data/user_profile_null_create.json

```diff
@@ -256,7 +256,7 @@
       "last_modified": "1970-01-01T00:00:00Z",
       "created": "1970-01-01T00:00:00Z",
       "verified" : true,
-      "display": null
+      "display": "staff"
```

Comment:
this should stay null for first/last name no?
or at least something non-staff - im guessing default is staff for staff and private for non-staff

Reply:
Yes. I think we settled on making this the skeleton for staff. I'm still in favor of dumping thins on the publishers :P

Do we wanna keep this for now or do you wanna put logic into the publishers? Or should I create different skeletons?

My main goal is to have phone book parity import for this week.